src/.vuepress/public/editor.html

```diff
@@ -780,14 +788,17 @@ <h5>Documentation</h5>
         if (optionImportMemory.checked) {
           options.push('--importMemory')
         }
+        if (optionZeroFilledMemory.checked) {
+          options.push('--zeroFilledMemory')
+        }
         if (optionExportTable.checked) {
           options.push('--exportTable')
         }
         if (optionImportTable.checked) {
           options.push('--importTable')
         }
-        if (optionExplicitStart.checked) {
-          options.push('--explicitStart')
+        if (optionExportStart.checked) {
+          options.push('--exportStart')
```

Comment:
I believe `--exportStart` takes a string argument, or is it working this way?

Reply:
Ah right! Will improve this

Reply:
I guess better just remove this option compleately

Reply:
Wdyt of keeping the option and adding a text field after, like `[ ] Export start [_start]` where `_start` is the default? Could imagine that it's useful for some use cases.

Reply:
I thought so too at first, but we can easily do this just by adding `import "wasi"` [see example in playground](https://www.assemblyscript.org/editor.html#IyFvcHRpbWl6ZT1zcGVlZCZydW50aW1lPXN0dWIKaW1wb3J0ICJ3YXNpIgojIWh0bWwKPHRleHRhcmVhIGlkPSJvdXRwdXQiIHN0eWxlPSJoZWlnaHQ6IDEwMCU7IHdpZHRoOiAxMDAlIiByZWFkb25seT48L3RleHRhcmVhPgo8c2NyaXB0IHR5cGU9Im1vZHVsZSI+CmNvbnN0IGV4cG9ydHMgPSBhd2FpdCBpbnN0YW50aWF0ZShhd2FpdCBjb21waWxlKCksIHsgLyogaW1wb3J0cyAqLyB9KQpjb25zdCBvdXRwdXQgPSBkb2N1bWVudC5nZXRFbGVtZW50QnlJZCgnb3V0cHV0JykKZm9yIChsZXQgaSA9IDA7IGkgPD0gMTA7ICsraSkgewogIG91dHB1dC52YWx1ZSArPSBgZmliKCR7aX0pID0gJHtleHBvcnRzLmZpYihpKX1cbmAKfQo8L3NjcmlwdD4=)

Reply:
It's a side-effect of `import "wasi"`, yeah, but doing so will also reconfigure the module, with another `abort` etc.

Reply:
Ok, done!